Make sure that _FILE_OFFSET_BITS is 64 to match gpgme.

--- configure.ac.orig	2013-04-24 11:13:23.985145198 +0200
+++ configure.ac	2013-04-24 11:13:34.536065452 +0200
@@ -24,6 +24,8 @@ AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_HEADERS([config.h])
 AC_DEFINE_UNQUOTED(BALSA_VERSION, "AC_PACKAGE_VERSION",[Defines the Balsa version])
 
+AC_SYS_LARGEFILE
+
 dnl #####################################################################
 dnl 2. Options
 dnl #####################################################################
